configure.ac

```diff
@@ -75,6 +75,12 @@ AC_ARG_WITH(stage1-default-location,
             [RKT_STAGE1_DEFAULT_LOCATION="${withval}"],
             [RKT_STAGE1_DEFAULT_LOCATION=auto])
 
+AC_ARG_WITH(stage1-default-images-directory,
+            [AS_HELP_STRING([--with-stage1-default-images-directory],
+                            [default directory for stage1 images, default: LIBDIR/rkt/stage1-images])],
```

Comment:
I'm not sure I should care what Fedora is using. Distros can't decide on a directory for arch specific binary data, for some it is `/usr/lib/rkt`, for other this will be `/usr/libexec/rkt`, some other will choose something else.

Also, we store system config in /usr/lib/rkt already, so I thought it might be a good idea to reuse the directory.